lualib/resty/kong/tls.lua

```diff
@@ -27,6 +27,9 @@ if ngx.config.subsystem == "http" then
     int ngx_http_lua_kong_ffi_get_full_client_certificate_chain(
         ngx_http_request_t *r, char *buf, size_t *buf_len);
     const char *ngx_http_lua_kong_ffi_disable_session_reuse(ngx_http_request_t *r);
+    // STACK_OF(X509)
```

Comment:
This line is unnecessary.

Reply:
It's a hint of the type of the opaque pointer.

Reply:
I checked the source code of lua-resty-core, there is no comments like this.

I think we need not care about the pointer type in Lua land.

Reply:
Actually, I copied this style from `lua-resty-openssl`😂. For example, [here](https://github.com/fffonion/lua-resty-openssl/blob/master/lib/resty/openssl/include/x509_vfy.lua#L17&L31). If this is not preferred, we can remove it.